### ISSUE TITLE
epub_use_index instead of html_use_index for epub builder

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1106]
The epub builder looks for the value of html_use_index in the config. All other config options have epub instead of html variants. For clarity and uniformity, this could be done for this config option as well

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1106
- Originally reported by: Theo J.A. de Vries
- Originally created at: 2013-02-12T21:08:49.473

**Comment by shimizukawa** [comment]
_From Roland Meister on 2013-02-14 21:10:04+00:00_

Do you have any special use case for a distinct epub_use_index setting?
The EpubBuilder class is a subclass of the StandaloneHTMLBuilder
class and inherits many settings from its parent class, not only
html_use_index (although that explicitly occurs in the source for the
epub builder).

I usually only make a distinct epub_ configuration setting if there is
an advantage in divergent settings for html and epub.

**Comment by shimizukawa** [comment]
_From Theo J.A. de Vries on 2013-02-15 16:33:00+00:00_

Well, I would like to have an epub without index and a html with index, from the same config file. So I guessed that there would be an epub_use_index, but there was not. Checked out the source, and found that this one was the only config variable in the epub builder that was not introduced with epub_. I can perfectly do without (I now make separate config files), but as indicated, for clarity and uniformity, I suggest to rename it. No big deal though.

Btw, thanks for a great piece of software!
Theo
